core/templates/pages/about-page/about-page.component.html

```diff
@@ -135,8 +135,8 @@ <h2 [innerHTML]="'I18N_ABOUT_PAGE_CREDITS_TAB_HEADING' | translate"></h2>
       <div class="oppia-static-extra-content oppia-about-clearfix">
         <div class="oppia-about-otter-container">
           <picture>
-            <source type="image/webp" [src]="getStaticImageUrl('/general/about_page_mascot.webp')">
-            <source type="image/png" [src]="getStaticImageUrl('/general/about_page_mascot.png')">
+            <source type="image/webp" [srcset]="getStaticImageUrl('/general/about_page_mascot.webp')">
```

Comment:
Why are these changes needed? Do we need to have a comment or a test to make sure this doesn't change if the change is necessary?